lib/createJestRunner.js

```diff
@@ -73,7 +73,17 @@ const createRunner = (runPath, { getExtraOptions } = {}) => {
                   return runner(baseOptions);
                 });
               })
-              .then(result => onResult(test, result))
+              .then(testResult => {
+                if (Array.isArray(testResult)) {
+                  testResult.forEach(result =>
+                    result.numFailingTests > 0
+                      ? onFailure(test, new Error(result.errorMessage))
```

Comment:
is this correct? I thought `onResult` was called with all test results regardless of whether the result is a failure or not, and `onFailure` was called if the test framework errored out. That might be wrong though, I haven't looked into the source code  of this module for some time 😅 

Could you show an example of how this is used in a runner and how it looks when used in Jest?

Reply:
You are correct, onFailure is when the framework errors out. It is passed a SerializableError https://github.com/facebook/jest/blob/f0dc9932cba828a1c51ef842c00c1ca51b7f2796/packages/jest-types/src/TestResult.ts#L31

I am working on this [jest-python](https://github.com/KaiSpencer/jest-python). It would be good to see if it was at all possible to simplify its implementation with this library.